test/worlds/tracked_vehicle_simple.sdf

```diff
@@ -0,0 +1,1626 @@
+                    <meta>
+                        <layer>0</layer>
+                    </meta>
+                </visual>
+                <collision name='Stairs_3_Collision_19'>
+                    <geometry>
+                        <box>
+                            <size>1 0.175 0.075</size>
+                        </box>
+                    </geometry>
+                    <pose>0 -1.6625 1.4625 0 0 0</pose>
+                </collision>
+            </link>
+            <static>1</static>
+        </model>
+        <model name='cylinder_obstacle'>
+            <static>1</static>
+            <pose>0 0 -0.06 0 0 1.5708</pose>
+            <link name='cylinder'>
+                <collision name='cylinder_collision'>
+                    <pose>0 0 0 0 1.5708 0</pose>
+                    <geometry>
+                        <cylinder>
+                            <radius>0.15</radius>
+                            <length>0.8</length>
+                        </cylinder>
+                    </geometry>
+                    <surface>
+                        <friction>
+                            <ode>
+                                <mu>1</mu>
+                            </ode>
+                        </friction>
+                    </surface>
+                </collision>
+                <visual name='cylinder_visual'>
+                    <pose>0 0 0 0 1.5708 0</pose>
+                    <geometry>
+                        <cylinder>
+                            <radius>0.15</radius>
+                            <length>0.8</length>
+                        </cylinder>
+                    </geometry>
+                </visual>
+            </link>
+        </model>
+        <model name='box_obstacle'>
+            <static>1</static>
+            <pose>2 2 0.028 1.7821 0 1.5708</pose>
+            <link name='box'>
+                <collision name='box_collision'>
+                    <pose>0 0 0 0 0 0</pose>
+                    <geometry>
+                        <box>
+                            <size>0.85 0.15 0.5</size>
+                        </box>
+                    </geometry>
+                    <surface>
+                        <friction>
+                            <ode>
+                                <mu>1</mu>
+                            </ode>
+                        </friction>
+                    </surface>
+                </collision>
+                <visual name='box_visual'>
+                    <pose>0 0 0 0 0 0</pose>
+                    <geometry>
+                        <box>
+                            <size>0.85 0.15 0.5</size>
+                        </box>
+                    </geometry>
+                </visual>
+            </link>
+        </model>
+        <include>
+            <name>pallet</name>
+            <pose>2 -2 0.1 0 0 0</pose>
+            <uri>https://fuel.ignitionrobotics.org/1.0/openrobotics/models/Euro pallet</uri>
+        </include>
```

Comment:
Can this `<include>` be removed? I see the model in the scene but it's not being used by the test. Seems like an unnecessary dependence on Fuel servers.

Reply:
I've also noticed this yesterday. I removed the pallet.